Types.h

```diff
@@ -5,6 +5,35 @@
 
 #include <MY17_Can_Library.h>
 
+// All names in the enum are taken from the NDrive manual
+typedef enum {
```

Comment:
actually i decided i rather like the increment-to-step feature so imma leave this enum there.